{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat OpenShift Container Platform 3.11",
      "fix_state": "Affected",
      "package_name": "atomic-openshift",
      "cpe": "cpe:/a:redhat:openshift:3.11"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "openshift",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "openshift4/ose-docker-builder",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "openshift4/ose-hyperkube",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "openshift-clients",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat Storage 3",
      "fix_state": "Not affected",
      "package_name": "heketi",
      "cpe": "cpe:/a:redhat:storage:3"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2020-10-14T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2020-8564 kubernetes: Docker config secrets leaked when file is malformed and loglevel \u003e= 4",
    "id": "1886637",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1886637"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "5.3",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:H/I:N/A:N",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-117",
  "statement": "",
  "acknowledgement": "Red Hat would like to thank the Kubernetes Product Security Committee for reporting this issue. Upstream acknowledges Nikolaos Moraitis (Red Hat) as the original reporter.",
  "name": "CVE-2020-8564",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": [
    "https://groups.google.com/g/kubernetes-announce/c/ScdmyORnPDk\nhttps://github.com/kubernetes/kubernetes/issues/95622"
  ]
}